src/gadgets/transaction.rs

```diff
@@ -10,47 +8,23 @@ use value::AllocatedValue;
 pub fn fill_cs<CS: ConstraintSystem>(
```

Comment:
bikeshed for later PR: would be cool to rename `spacesuit::transaction::fill_cs()` into `spacesuit::cloak()` and likewise, `spacesuit::<gadget>::fill_cs()` into `spacesuit::<gadget>()`. We can keep the module organization as-is, though, just not expose it to the user.

Reply:
makes sense! 👍